src/lib/getSchemaUi.ts

```
@@ -0,0 +1,228 @@
+
+  static createCircularReferenceProperty(name: string, circularRef: string): OAProperty {
+    return {
+      name,
+      types: ['object'],
+      required: false,
+      description: `Circular reference to **${circularRef}**`,
+      meta: { isCircularReference: true },
+    }
+  }
+
+  static createOneOfProperty(oneOfProperties: Partial<OpenAPI.SchemaObject>[], name: string = ''): OAProperty {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)